doc/tutorials/index.md

```diff
@@ -63,11 +101,59 @@ In this section we will review different approaches for styling components, from
 :hidden:
 :maxdepth: 2
 
+beginner/panel_serve
+beginner/develop_editor
+beginner/develop_notebook
+beginner/display_pn_panel
+beginner/display_panes
 param
-components
 interactivity
 development
 layouts
 structure
 styling
 ```
+
+---
+
+## Intermediate
+
+These tutorials are for you that is ready to navigate and explore more advanced features of Panel.
+
+You will learn to develop and deploy complex tools and multi-page apps that can support more demanding use cases.
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`browser;2.5em;sd-mr-1` Serve Panel Apps
+:link: intermediate/panel_serve
+:link-type: doc
+
+Learn to serve multi-page apps and how-to configure `panel serve`.
+:::
+
+:::{grid-item-card} {octicon}`code;2.5em;sd-mr-1` Develop in an Editor
+:link: intermediate/develop_editor
+:link-type: doc
+
+Learn more advanced features like debugging to develop efficiently with Panel in an editor
+:::
+
+::::
+
+```{toctree}
```

Comment:
I need to figure out if its possible to seperate toctrees in a single document. And how.

![image](https://github.com/holoviz/panel/assets/42288570/63006df0-de20-468d-a909-be0ad447a79d)

Reply:
We tried using `:caption:`. It works in alabaster theme. But not the theme we are using.